test/test_model.py

```diff
@@ -0,0 +1,4 @@
+from src.Model import Model
+
+def test_model():
+  assert(Model.evaluate(Model)[0] == 0.4)
```

Comment:
You aren't guaranteed that the model will always evaluate the first action value to 0.4. I think you should instead add an assertion about the 
- shape of the result 
- the type of data
- or the bounds

You should also avoid passing in model to evaluate like that. Instead do,
```python
model = Model(image, height, width)
# Then use model.evaluate() with your assertions
```